README.md

```diff
@@ -171,7 +171,15 @@ For any questions regarding secvarctl, feel free to reach out: [Nick Child](nick
 		-v , verbose, gives process info
 		-n <varName> , name of secure boot variable, used when generating an auth file, PKCS7, or when the input file contains hashed data rather than x509 (use '-n dbx'), current <varName> are: {'PK','KEK','db','dbx'}
 		-f force generation, skips validation of input file, assumes format to be correct
-		-t <time> , where time is of the format 'y-m-d h:m:s'. creates a custom timestamp used when generating an auth or PKCS7 file, if not given then current time is used
+		-t <time> , where <time> is of the format described below. creates a custom timestamp used when generating an auth or PKCS7 file, if not given then current time is used, all times are in UTC
+                    format of <time> = 'YYYY-MM-DDThh:mm:ss' where:
+                        - 'T' appears literally
+                        - 'YYYY' four-digit year
+                        - 'MM' two-digit month (01=January, etc.)
+                        - 'DD' two-digit day of month (01 through 31)
```

Comment:
Move `T` to be after `DD`, so the bulletpoint order matches the order in the format sequence